templates/configmap.yaml

```diff
@@ -159,7 +159,7 @@ data:
             "kubernetes_pod_operator": "False"
         }
 
-        if 'airflow-worker' in pod.labels.keys() or \
+        if 'airflow-worker' in pod.labels.keys() and \
```

Comment:
The problem with this is the old Airflow versions didn't have 'airflow-worker' label IIRC, might have to check.

Reply:
I went back to 1.10.10, I can go further though.

Reply:
It's in 1.10.0, so I think we should be good.